packages/itwinui-css/backstop/tests/table.html

```diff
@@ -1253,46 +1253,46 @@ <h2>Default</h2>
             </div>
           </div>
 
-          <div class="iui-row iui-negative">
-            <div class="iui-cell iui-slot">
+          <div class="iui-table-row iui-negative">
```

Comment:
We could use `data-iui-status` on both rows and cells.
```suggestion
          <div class="iui-table-row" data-iui-status="negative">
```

Reply:
`iui-positive/negative/warning` has been replaced with `data-iui-status='positive/negative/warning'`